screens/LoadingScreen.tsx

```diff
@@ -15,11 +21,37 @@ export default function LoadingScreen({
     return null;
   }
   return (
-    <View style={styles.container}>
+    <LinearGradient
+      colors={
+        isLight
+          ? [gradientLight.from, gradientLight.to]
+          : [gradientDark.from, gradientDark.to]
+      }
+      style={styles.container}
+    >
       <Text style={styles.title}>Prilla</Text>
       <Text style={styles.slogan}>GOTTA SNUS THEM ALL</Text>
       <View style={styles.separator} lightColor="#eee" darkColor="#eee" />
-    </View>
+
+      <TouchableOpacity
+        style={styles.button}
```

Comment:
Why does the loadingScreen have functionality for register and log in? Shouldnt it just have a loading spinner?